src/libraries/System.Net.NetworkInformation/src/System/Net/NetworkInformation/StringParsingHelpers.Statistics.cs

```diff
@@ -407,16 +407,15 @@ internal static IPInterfaceStatisticsTable ParseInterfaceStatisticsTableFromFile
             {
                 sr.ReadLine();
                 sr.ReadLine();
-                int index = 0;
                 while (!sr.EndOfStream)
                 {
                     string line = sr.ReadLine()!;
                     if (line.Contains(name))
                     {
```

Comment:
Does it make sense to remove the Contains here since we're checking with SequenceEqual later again?

Reply:
I thought the `Contains` is cheap so let it do a precheck might have a better performance.